DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db

```diff
@@ -339,3 +339,44 @@ record(ao, "$(P)RAW:SP")
     field(PINI, "$(SP_PINI)")
     info(autosaveFields, "VAL")
 }
+
+record(ao, "$(P)SIM:RAW"){
```

Comment:
I've just realised what the problem is, in the act of adding in the RAW readback, I failed to update these aliases correctly, they should be the other way around.